entc/gen/type.go

```diff
@@ -932,7 +963,7 @@ func aliases(g *Graph) {
 			}
 			name := f.Type.PkgName
 			if name == "" && f.Type.PkgPath != "" {
-				name = path.Base(f.Type.PkgPath)
+				name = importPathToAssumedName(f.Type.PkgPath)
```

Comment:
Can't we extract it from the Go identifier, instead of parsing the import path? e.g. `pkg.type` => `pkg`. Because import path does not always reflect the package name.

```go
import (
    "a"   // a.T
    "b/a" // b.T
)
```

The above import is valid if the 2 packages have different names.